uint8 id

bool truth
float32 radius_truth
float32 x_truth
float32 y_truth

bool estimate
float32 radius_estimate
float32 x_estimate
float32 y_estimate

float32 estimate_euclidean_error

float32 warning_delay
float32 probe_time
